servicex_app/servicex/code_gen_adapter.py

```diff
@@ -36,8 +36,7 @@ def __init__(self, code_gen_url, transformer_manager):
         self.transformer_manager = transformer_manager
 
     def generate_code_for_selection(
-        self, request_record: TransformRequest, namespace: str
-    ):
+            self, request_record: TransformRequest, namespace: str, transformer_image: str = None):
         """
```

Comment:
Need to update comment to reflect new argument

Reply:
Added new :param comment to reflect the new parameter in function